core/kits/documents/tabs/settings-layout.php

```diff
@@ -105,21 +106,20 @@ protected function register_tab_controls() {
 		$this->add_control(
 			'space_between_widgets',
 			[
-				'label' => $widgets_space_label . ' (px)',
+				'label' => $widgets_space_label,
 				'type' => Controls_Manager::SLIDER,
-				'default' => [
-					'size' => 20,
-				],
+				'size_units' => [ 'px', 'em', '%', 'rem', 'vw', 'custom' ],
 				'range' => [
 					'px' => [
 						'min' => 0,
 						'max' => 40,
 					],
 				],
 				'placeholder' => [
-					'size' => '20',
+					'size' => 20,
+					'unit' => 'px',
```

Comment:
Remove the units. It's not required. Focus mainly on `size_units` in this PR.

Reply:
Done. Normally everything is good, sorry for any confusion, I wanted to make something really complete, without knowing more about the architecture of the code regarding the units.